docs/configuration.md

```diff
@@ -117,8 +116,8 @@ Each resolver must be defined as a string in following format: `[net:]host:[port
 The `commonName` parameter overrides the expected certificate common name value used for verification.
 
 !!! note
-    Blocky needs at least the configuration of the **default** group with at least one upstream DNS server. This group will be used as a fallback, if no client
```

Comment:
This indentation is important otherwise the note block is broken. You can test documentation by executing `make serve_docs`

Reply:
thx, fixed it